hipchat.go

```diff
@@ -86,15 +86,22 @@ type ErrorResponse struct {
 type Client struct {
 	AuthToken string
 	BaseURL   string
+	ProxyURL  string
```

Comment:
Proxy settings can now be set via setting the Transport property after creating the client. This library shouldn’t care about proxies specifically.
```Go
c := hipchat.NewClient("<PUT YOUR AUTH TOKEN HERE>")
c.Transport = &http.Transport{Proxy: http.ProxyURL("<PROXY URL>")}
```